script.js

```diff
@@ -1,8 +1,24 @@
 const steps = d3.selectAll(".step");
 const svg = d3.select("svg");
+const graphic = d3.select(".graphic");
 const width = svg.node().clientWidth;
 const height = svg.node().clientHeight;
 
+const spuddyText = [
+  "Hi there! My name is Spuddy! I'll be your personal potato buddy today! Try scrolling to see some visualizations, and be sure to click on me for more info!",
```

Comment:
<img width="396" alt="image" src="https://github.com/user-attachments/assets/448e0816-d3be-4773-82c2-1156029a028f">
Is a bit too long on my screen